addons/website_livechat/controllers/main.py

```diff
@@ -36,7 +36,7 @@ def channel_rating(self, channel, **kw):
         ratings_per_partner = {partner_id: dict(great=0, okay=0, bad=0)
                                for partner_id in ratings.mapped('rated_partner_id.id')}
         total_ratings_per_partner = dict.fromkeys(ratings.mapped('rated_partner_id.id'), 0)
-        rating_texts = {10: 'great', 5: 'okay', 1: 'bad'}
+        rating_texts = {5: 'great', 3: 'okay', 1: 'bad'}
```

Comment:
Shouldn't we be backward compatible ?

Reply:
nope, because this is actually a fix. It doesn't work anyway with old values.

Reply:
Then adding a backward compatible for 10 (if we receive 10 set to 5, otherwise keep as it is) ?